pages/pipelines/cluster_queue_metrics.md

```diff
@@ -2,6 +2,9 @@
 
 Queue metrics show the most important statistics to help you optimize your agent setup and monitor a queue's performance. These statistics are updated on the page every 10 seconds.
 
+> 📘
+> `Unclustered` agents are not reported in advanced queue metrics.
```

Comment:
Thanks very much @mcncl for your contribution! 🎉 

A couple of suggestions, 'unclustered agents' is a key term in this context and should be formatted as such in our revised [writing style guide on key terms](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#key-terms-and-emphasis).

Also, I think it'd help at this point to provide a convenient link for readers if they want to find out more about what unclustered agents (and their tokens) are.

Hence, these suggested changes...

```suggestion
> _Unclustered agents_ are not reported in advanced queue metrics. Learn more about unclustered agents in [Unclustered agent tokens](/docs/agent/v3/unclustered-tokens).
```

Reply:
Committed your suggestions @gilesgas